compiler/AST/baseAST.cpp

```diff
@@ -241,11 +223,17 @@ void destroyAst() {
 
 void
 verify() {
+  verifyRemovedIterResumeGotos();
+  verifyCopiedIterResumeGotos();
+
   #define verify_gvec(type)                       \
     forv_Vec(type, ast, g##type##s) {             \
+     if (isAlive(ast)) {                          \
       ast->verify();                              \
+     }                                            \
     }
   foreach_ast(verify_gvec);
+  rootModule->verify();  // rootModule is not isAlive()
```

Comment:
Add more detail to this comment to be in line with your PR, mentioning that it's alive but fails isAlive().